requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy>=1.22
+scipy>=1.10
+numba>=0.59.1
+pyFFTW>=0.15.0
```

Comment:
Can we remove this file  completely now?

Reply:
Yes, and I think I should because it does not reflect the FFTW dependency.